examples/streaming-big-dataset/big_data_service.bal

```diff
@@ -0,0 +1,46 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+import ballerina/mysql;
+
+// Create mysql client endpoint.
+mysql:Client testDB = new({
+        host: "localhost",
+        port: 3306,
+        name: "StreamTestDB",
+        username: "user1",
+        password: "pass1",
+        poolOptions: { maximumPoolSize: 5 }
+    });
+
+// Create a HTTP service and bind to a listener on port 9090
+service dataService on new http:Listener(9090) {
+
+    resource function getData(http:Caller caller, http:Request req) {
+        http:Response res = new;
+
+        // Invoke `select` action on the mysql database endpoint and obtain a large amount of data.
+        var selectRet = testDB->select("SELECT * FROM Data", ());
+        if (selectRet is table <record {}>) {
+            // Convert the obtained data to JSON. Please note that this conversion would not load
```

Comment:
Shall we mention in the comment that , similarly we can convert to XML?

Reply:
+1. Will update